src/goose.rs

```diff
@@ -1510,6 +1511,7 @@ impl GooseUser {
 
         // Once past the throttle, the request is officially started.
         let started = Instant::now();
+        let request_time = Utc::now();
```

Comment:
I don't understand what `Utc::now()` is giving us that `Instant::now()` isn't? It feels like we should use one or the other, but not both?

Reply:
Reworked as we discussed so this is now gone.